### PR TITLE
Import biogo.graphics palettes

### PR DESCRIPTION
**Pull request by kortschak**
Minor changes to packages in that repo:

* Change from gocheck to testing.
* golint changes.

All AUTHORS of this code are already in gonum/A+C (me and my employer). biogo copyright notices are retained purely for historical reasons.

### CODE REVIEW

File: palette/brewer/brewer.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+// The colors defined here are from http://www.ColorBrewer.org/ by Cynthia A. Brewer,
+// Geography, Pennsylvania State University.
+//
+// For more information see:
+// http://www.personal.psu.edu/cab38/ColorBrewer/ColorBrewer_learnMore.html
+//
+package brewer
+
+import (
+	"code.google.com/p/biogo.graphics/palette"
```
Comment (sbinet) [inline palette/brewer/brewer.go]: shouldn't this be `github.com/gonum/plot/palette` ?

Comment (kortschak) [inline palette/brewer/brewer.go]: It should.

**Comment by kortschak** [comment]
PTAL

File: palette/hsva_test.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import (
+	"image/color"
+	"testing"
+)
+
+func withinEpsilon(a, b, epsilon uint32) bool {
+	d := int64(a) - int64(b)
+	if d < 0 {
+		if d == -d {
+			panic("color: weird number overflow")
```
Comment (sbinet) [inline palette/hsva_test.go]: `s/color:/palette:/`
(or `plot`?)

**Comment by sbinet** [comment]
perhaps also add a simple example?

other than that: LGTM

**Comment by kortschak** [comment]
I can't think of any non-facile example, so I'll push this. If you want I can revisit this after discussion.

**Comment by eaburns** [comment]
Is it appropriate to use this in plotutil to replace the various color slices? If so, open an issue so that we don't forget.

**Comment by kortschak** [comment]
That can be done. I'll send a PR today.

**Comment by kortschak** [comment]
Actually, this requires further discussion since there are at least a couple of different approaches, so #11.

**Pull request merged**